docs/apache-airflow/authoring-and-scheduling/timetable.rst

```diff
@@ -239,7 +239,7 @@ triggered when ``catchup`` is ``True``.
 
 When ``catchup`` is ``False``, there is difference in how a new DAG run is triggered.
 
-`CronTriggerTimetable`_ and `CronDataIntervalTimetable`_ trigger DAG runs at the same time, because they were given the same CRON string. However, the timestamp for the ``run_id`` is different for each.
+`CronTriggerTimetable`_ and `CronDataIntervalTimetable`_ trigger DAG runs at the same time. However, the timestamp for the ``run_id`` is different for each.
 
 - `CronTriggerTimetable`_ has a ``run_id`` timestamp showing when DAG run started.
```

Comment:
One more nit of clarification. DAG Runs can be delayed but those delays don't change the run_id.
```suggestion
- `CronTriggerTimetable`_ has a ``run_id`` timestamp showing when DAG run is able to start.
```